java/grpc-client/src/main/java/io/vitess/client/grpc/netty/DefaultChannelBuilderProvider.java

```diff
@@ -0,0 +1,27 @@
+package io.vitess.client.grpc.netty;
+
+import io.grpc.netty.NettyChannelBuilder;
+import io.netty.channel.EventLoopGroup;
+import io.netty.channel.nio.NioEventLoopGroup;
+import io.netty.util.concurrent.DefaultThreadFactory;
+import io.vitess.client.grpc.RetryingInterceptor;
+import io.vitess.client.grpc.RetryingInterceptorConfig;
+
+public class DefaultChannelBuilderProvider implements NettyChannelBuilderProvider {
+  private static final EventLoopGroup ELG = new NioEventLoopGroup(
```

Comment:
We should probably exclude the ELG change in the default provider as this deviates from former behavior